spec.md

```diff
@@ -119,16 +119,34 @@ include information about the occurrence, details about the data that was
 changed, or more. See the [Event Data](#event-data) section for more
 information.
 
+#### Event Format
+
+An Event Format specifies how to serialize a CloudEvent as a sequence of bytes.
+Stand-alone event formats, such as the [JSON format](json-format.md), specify
+serialization independent of any protocol or storage medium.  Protocol Bindings
+MAY define formats that are dependent on the protocol.
+
 #### Message
 
 Events are transported from a source to a destination via messages.
 
+A "structured-mode message" is one where the event is fully encoded using
```

Comment:
regardless of where we land on this PR - I like adding these definitions to the spec itself.

Reply:
Yep, I find them very counter-intuitive so we should write them down somewhere obvious.